gc_little_helper_II.user.js

```diff
@@ -4195,31 +4195,37 @@ var mainGC = function() {
+        if ($('#gclh_linkAdditionalInfo.working')[0]) return;
+        $('#gclh_linkAdditionalInfo').addClass('working');
         var anzLines = $('table.Table tbody tr').length / 2;
         if ($('table.Table tbody tr').first().find('td:nth-child(4)').find('img[src*="WptTypes"]')[0]) var colGccode = 3;
         else var colGccode = 4;
         var colName = colGccode + 1;
         $('table.Table tbody tr').each(function() {
             if ($(this).find('td:nth-child('+colGccode+') a')[0]) {
                 var gccode = $(this).find('td:nth-child('+colGccode+') a')[0].innerHTML;
-                if (!$('#gclh_colCorrCoords')[0]) $(this).find('td:nth-child('+colName+')').after('<td id="cc_'+gccode+'" class="cc_cell"></td>');
+                if (!$('#gclh_colAdditionalInfo')[0]) $(this).find('td:nth-child('+colName+')').after('<td id="cc_'+gccode+'" class="cc_cell"></td>');
                 else $('#cc_'+gccode)[0].innerHTML = "";
             } else {
-                if (!$('#gclh_colCorrCoords')[0]) $(this).find('td:nth-child(2)').after('<td></td>');
+                if (!$('#gclh_colAdditionalInfo')[0]) $(this).find('td:nth-child(2)').after('<td></td>');
                 return;
             }
             $.get('https://www.geocaching.com/geocache/'+gccode, null, function(text){
                 var corr_gccode = $(text).find('#ctl00_ContentBody_CoordInfoLinkControl1_uxCoordInfoCode')[0].innerHTML;
+                // coorected coords
                 if (text.includes('"isUserDefined":true,"newLatLng"')) $('#cc_'+corr_gccode)[0].innerHTML = '<img title="Corrected Coordinates" alt="Corr. Coords" src="'+global_green_tick+'">';
                 else $('#cc_'+corr_gccode)[0].innerHTML = '<img style="opacity: 0.8;" title="No Corrected Coordinates" alt="No Corr. Coords" src="'+global_red_tick+'">';
+                var diff = $(text).find("#ctl00_ContentBody_uxLegendScale img").attr("alt");
+                diff = diff.substr(0,diff.indexOf(' '));
+                var terr = $(text).find("#ctl00_ContentBody_Localize12 img").attr("alt");
+                terr = terr.substr(0,terr.indexOf(' '));
+                $('#cc_'+corr_gccode)[0].innerHTML = $('#cc_'+corr_gccode)[0].innerHTML + ' - D'+diff+'/T'+terr;
                 anzLines--;
-                if (anzLines == 0) $('#gclh_linkCorrCoords').removeClass('working');
+                if (anzLines == 0) $('#gclh_linkAdditionalInfo').removeClass('working');
             });
         });
-        if (!$('#gclh_colCorrCoords')[0]) $('table.Table thead tr th:nth-child('+colName+')').after('<th id="gclh_colCorrCoords" style="width: 90px;"><span title="Caches with Corrected Coordinates">Corr. Coords</span></th>');
+        if (!$('#gclh_colAdditionalInfo')[0]) $('table.Table thead tr th:nth-child('+colName+')').after('<th id="gclh_colAdditionalInfo" style="width: 90px;"><span title="Additional Information (Corrected Coordinates/ Diff-Terr)">Corr. Coords - D/T</span></th>');
```

Comment:
- Spalte etwas verbreitern damit es keine Umbrüche in den Zeilen gibt. Damit Icon GC Tour nicht umbricht abhängig machen von setting page-view: 
if (settings_new_width >= 1050) css += "#gclh_colAdditionalInfo {width: 122px;}";
else css += "#gclh_colAdditionalInfo {width: 92px;}";
- title="Additional information (Corrected Coordinates - Difficulty/Terrain)" 
können wir doch ausschreiben und genauso machen wie im Button
- Corr.Coords - D/T 
anstatt Corr. Coords - D/T, also blank weglassen, spart etwas Platz